polygeist/lib/Dialect/Polygeist/Transforms/BareMemRefToLLVM.cpp

```diff
@@ -635,6 +748,128 @@ struct StoreMemRefOpLoweringOld : public MemAccessLowering {
     return success();
   }
 };
+
+struct ViewMemRefOpLoweringOld : public ConvertOpToLLVMPattern<memref::ViewOp> {
```

Comment:
Most code are the same as `ViewMemRefOpLowering`, except, there are some bitcast.